wasp/boards/simulator/watch.py

```diff
@@ -5,6 +5,9 @@
 def sleep_ms(ms):
     time.sleep(ms / 1000)
 time.sleep_ms = sleep_ms
+time.ticks_ms = lambda : int(time.time() * 1000)
+time.ticks_us = lambda : int(time.time() * 1000 * 1000)
+time.ticks_diff = lambda x, y : x-y
```

Comment:
Looks like these are a bit more complex than simple subtraction and multiplication: https://docs.micropython.org/en/latest/library/time.html#time.ticks_diff https://docs.micropython.org/en/latest/library/time.html#time.ticks_ms

Reply:
Nevermind, that's just wraparound, which shouldn't be a problem with python's arbitrary-precision integers